doc/authoring_tests.md

```diff
@@ -36,6 +36,13 @@ It is a requirement for the command owner to maintain their test in live mode.
 * Name test methods in the following format: `test_<module>_<feature>`.
 * The scenario test must be able to run repeatedly in live mode. The feature owner is responsible of maintaining their scenario tests.
 
+### Scenario Test Best Practice
+
+* 100% commands coverage of module: Scenerio tests __MUST__ cover all commands in the module except the `wait` commands. Please use `azdev cmdcov {module}` to check command coverage of module.
+* 100% examples coverage of command: Scenerio tests __MUST__ coverage all examples of the command.
+* 100% arguments coverage of command: Scenerio tests should coverage all arguments in the command. Please use `azdev cmdcov {module} --level argument` to check argument coverage.
```

Comment:
```suggestion
* 100% arguments coverage of command: Scenerio tests should coverage all arguments in the command. Please use `azdev cmdcov {module} --level argument` to check argument coverage. If cannot add a test because of some reason, please add missing_parameter_test_coverage in the linter_exclusions.yml.
```